### DISCUSSION TITLE
I am not able to inspect notification dialog box from developer tools

### DISCUSSION THREAD

**Discussion by inchara-1998** [microsoft/vscode-discussions#734] [Extension Development QnA] [answered]
I am not able to inspect notification dialog box from developer tools.
Is it possible to do that or not. we are working on automation for our extension need to check can we see any html related code for that dialog box.

![image](https://github.com/microsoft/vscode-discussions/assets/118963451/f68347ee-35a8-414b-ad2f-3363a7aca96a)

**Comment by r3m0t** [comment, marked as answer]
The notification is shown using Electron APIs, it doesn't have any HTML.

You can apply the setting `"window.dialogStyle": "custom"` to change this.

**Comment by inchara-1998** [comment]
means do we need to set this property ??
can you give any example how i can use this

**Reply by r3m0t** [reply]
Run the command `Open Settings (JSON)` and paste the setting in.
